root/defaults/nginx/site-confs/default.conf.sample

```diff
@@ -18,22 +18,9 @@ server {
     index index.html index.htm index.php;
 
     # enable subfolder method reverse proxy confs
-    include /config/nginx/proxy-confs/*.subfolder.conf;
+    include /config/nginx/subfolder-confs/*.subfolder.conf;
```

Comment:
Should it be just `*.conf`?

Reply:
Should be resolved in daf99713a97f809748a67d6c3ec40ed1e1fcfe24